__tests__/app.test.js

```diff
@@ -86,15 +86,33 @@ describe("/api/articles", () => {
         });
       });
   });
-  test("GET 404: responds with an error when wrong endpoint is requested", () => {
+  test("GET 200: responds with an array of all articles filtered by topic if topic query value is specified", () => {
     return request(app)
-      .get("/api/banana")
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then((response) => {
+        response.body.articles.forEach((article) => {
+          expect(article.topic).toBe("mitch");
+        });
+      });
+  });
+  test("GET 404: responds with an error if no articles are found for topic query value", () => {
+    return request(app)
+      .get("/api/articles?topic=7777")
       .expect(404)
       .then((response) => {
-        expect(response.body.msg).toBe("Invalid path");
+        expect(response.body.msg).toBe("No articles found");
       });
   });
 });
+test("GET 404: responds with an error when wrong endpoint is requested", () => {
```

Comment:
This bad path test isn't related to the current test and is actually covering all bad paths so should be in its own describe block, or if we already have a similar test then this one won't be needed. If a test auto passes it's a good indicator that we may have written a bad test.

Reply:
Have removed this test as there is already an appropriate test block for it